### DISCUSSION TITLE
Can workspaces and displays be ,,decoupled''?

### DISCUSSION THREAD

**Discussion by bzdula** [i3/i3#5011] [Q&A] [answered]
Hi, is there a somewhat trivial way to have a full 1-9 workspaces numbered separately for each monitor ? So I can for instance have Mod+number go to workspace on current monitor

**Comment by orestisfl** [comment, marked as answer]
Short answer: no

Long answer: yes, if you accept that this workspaces will not be named 1-9. The way to do is to bind `$mod+1...9` to an [IPC script](https://i3wm.org/docs/ipc.html) that a. checks currently active monitor b. focuses/opens the workspace with the correct number. Since workspaces need unique names you'll need to name the wokrspaces something like `1: left` and `1: right` or whatever naming scheme you prefer.

**Reply by Airblader** [reply]
Small addition: It is possible to name them 1—9 on each output by assigning different numbers to them (eg 1-9, 10—20, ...) and using `strip_workspace_numbers` to hide the number and show only the name.
